### PR TITLE
Merge build-branch with master

### PR DESCRIPTION
**Pull request by ershadul1**
### Project Specification

- We used bootstrap wherever possible in this project.

- We tried to make the page look like the original.

- Made the page responsive to different screen sizes.

- A descriptive README.MD file to accompany and explain the project, complete with screenshots, and live demo.

- The project has passed linting with GitHub Actions and validation from w3c's markup validation service.

### CODE REVIEW

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  
:shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is amazing you master it GitHub Flow you have a professional README file. (Check to improve [section](#Toimprove))

### To change [MANDATORY]

**MEDIUM SIZE**

- [x] Medium screen size (tablet). Your approach is different from the original page.
  - In this size, your _FEATURES STORIES_ section should be on the left and the _TOP STORY_ on the right. Check something about **[order](https://getbootstrap.com/docs/4.0/layout/grid/#reordering)** with Bootstrap

image



**Footer**

- [x] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/)). This applies form md screen size to up
  - Copyright section should be next to the logo and also a line next to each link

image


### To improve [OPTIONAL]

- [x] README file, you try using just one image ([mockup](https://cdn1.vectorstock.com/i/1000x1000/10/15/laptop-smartphone-and-tablet-mockup-vector-13691015.jpg)) for three different sizes, or use just one by default.

- [x] Use a title for your pull request message and don't forget to use meaningful in all your commits messages.

  - Next time you might try using these templates:

    ```
    An unmeaningful title
    - Descriptive messages
    - More meaningful messages
    ```

    

image

- [ ] Try use a reset [CSS file](https://necolas.github.io/normalize.css/8.0.1/normalize.css), it's going to improve your styles. 

- [ ] Always use comments. One of the best practices for writing CSS is placing a comment on each group of elements. This will give you an ease of accessibility when you are looking for specific groups.

  Consider using this template
  
  ```css
  /* Project Meta */
  
  /* reset  */
  
  /* general */
  
  /* typography */
  
  /* grid */
  
  /* header */
  
  /* footer */
  
  /* Page template A */
  
  /* Page template B */
  
  /* Media Queries */
  ```
  
  
  
  **Bad practice**

```css
/* margin-top:23px; */
```



> Taken from [Good and Bad CSS practices](https://speckyboy.com/good-bad-css-practices/)



<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: index.html [hunk 1]
```diff
@@ -483,19 +483,19 @@ <h5 class="card-title">Inside the Mysterious Area 51: Aliens, UFOs, Conspiracies
...
-            <div class="magazine-card my-2 col-lg-3 col-md-3 col-sm-12">
+          <div class="row mt-4">
+            <div class="magazine-card my-2 col-lg-3 col-md-3 col-sm-12 align-items-baseline">
               <div class="card rounded-0 border-0 position-relative">
                 <div class="position-relative">
-                  <img src="/image/magazine-img-1.webp" class="card-img-top rounded-0" alt="tech academy">
+                  <img src="/image/magazine-img-1.jpg" class="card-img-top rounded-0" alt="tech academy">
                   <a href="#" class="category text-uppercase">july 17 issue</a>
                 </div>
               </div>
```
Comment (bertil291utn) [inline index.html]: **[Use semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)**

You still have to use `<section>` and `<article>` tags, use divs where is necessary.

**Comment by ershadul1** [comment]
Fixed issues according to the review.

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

### The project looks really good. Congratulations :+1: 

I also saw that you fixed the changes asked by my colleague above :pray: 
##### If you would like to improve the project a little, I'm suggesting a small change (it is optional):
- [optional] On the mobile screen size, there are spaces between sections that we can see the blue background.

| Original | Yours |
|:---:|:---:|
|Screen Shot 2020-07-10 at 13 55 03 | Screen Shot 2020-07-10 at 13 55 46 |

### Go ahead and merge! Good luck on the next project.

Good Luck and Happy Coding

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**